src/promptflow/tests/executor/e2etests/test_batch_engine.py

```diff
@@ -123,7 +125,7 @@ def test_batch_storage(self, dev_connections):
             ),
         ],
     )
-    def test_batch_run(self, flow_folder, inputs_mapping, dev_connections):
+    def test_batch_run(self, flow_folder, inputs_mapping, dev_connections, recording_injection):
```

Comment:
Can we use the "autouse" fixture to avoid adding `recording_injection` for each test case?
https://docs.pytest.org/en/7.1.x/how-to/fixtures.html#autouse-fixtures-fixtures-you-don-t-have-to-request